lib/phoenix/template.ex

```diff
@@ -199,14 +222,14 @@ defmodule Phoenix.Template do
       Returns the template root alongside all templates.
       """
       def __templates__ do
-        {@template_root, unquote(names)}
+        {@template_root, @lookup_pattern, unquote(names)}
```

Comment:
It was used here https://github.com/phoenixframework/phoenix/pull/1661/files#diff-a3b14cef44a70d6236e4c4c22d6fb30dR181
but think will be gone with upcoming changes